ReactNativeClient/lib/models/Folder.js

```diff
@@ -68,9 +68,10 @@ class Folder extends BaseItem {
 
 		if (options.deleteChildren) {
 			let noteIds = await Folder.noteIds(folderId);
-			for (let i = 0; i < noteIds.length; i++) {
-				await Note.delete(noteIds[i]);
-			}
+			// for (let i = 0; i < noteIds.length; i++) {
+			// 	await Note.delete(noteIds[i]);
+			// }
```

Comment:
remove

Reply:
I am closing this PR and making a fresh one

Reply:
For future development I'd suggest to learn git. git is more than `git pull` and `git push`.